backend/app/router/endpoints/questions.py

```
@@ -2,48 +2,80 @@
+    notes_path = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))),
+        "notes",
+        "Notes on Transforms d557c60e03b64da285174f30cb0e2c18.md"
+    )
+    with open(notes_path, "r", encoding="utf-8") as f:
+        trans_content = f.read()
+    
+    completion = client.beta.chat.completions.parse(
+    model="Cerebras-Llama-4-Maverick-17B-128E-Instruct",
+    messages=[
+        {"role": "system", "content": "You are a quiz creator. You create short answer questions (not multiselect) only. You will be given a document and you will need to create a quiz from it, wtith 3 questions."},
```

Pylint (3.3.7)

[convention] Line too long (215/100)

(C0301)